go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
+
+go 1.23.4
+
+require (
```

Comment:
You are using both v2 and v3 versions of some packages, like `github.com/caarlos0/env`. It's worth checking if you can align to a single major version across your dependencies to avoid compatibility issues and confusion.